bazaar/src/lib.rs

```diff
@@ -48,6 +48,7 @@ pub mod pallet {
 
 	#[pallet::pallet]
 	#[pallet::generate_store(pub (super) trait Store)]
+	#[pallet::without_storage_info]
 	pub struct Pallet<T>(PhantomData<T>);
```

Comment:
Substrate will make this forbidden in the future. By then, we need to have completely solved #132.